src/__internal__/input/input.style.ts

```diff
@@ -20,6 +22,9 @@ const StyledInput = styled.input<
     -webkit-background-clip: text;
   }
 
+  ${({ inputBorderRadius }) =>
+    `border-radius: ${inputBorderRadius || "var(--borderRadius050)"};`}
```

Comment:
**suggestion** (non-blocking): as above, this default could be handled better (imo) with a default to `inputBorderRadius` in the `Input` component itself